collector/aws/elbv2.go

```diff
@@ -203,32 +211,31 @@ func (el *ELBV2Manager) Detect() ([]DetectedELBV2, error) {
 }
 
 // GetPricingFilterInput prepare document elb pricing filter
-func (el *ELBV2Manager) GetPricingFilterInput() *pricing.GetProductsInput {
+func (el *ELBV2Manager) GetPricingFilterInput(loadbalancer *elbv2.LoadBalancer) *pricing.GetProductsInput {
+
+	var loadBalancerProductFamily string
```

Comment:
In this case you already know which type is it (you have a logic that take from `loadBalancerCWNameSpace `).
is incase loadBalancerCWNameSpace == map[string]string
you can change it to 
map[string]strcut
then you can remove this switch case